modules/utxo-lib/src/bitgo/UtxoPsbt.ts

```diff
@@ -414,7 +414,6 @@ export class UtxoPsbt<Tx extends UtxoTransaction<bigint>> extends Psbt {
 
   private getTaprootHashForSig(
     inputIndex: number,
-    forValidate: boolean,
```

Comment:
do you know why this was in the signature to begin with?

Reply:
@brandonblack ?

Reply:
Yeah, to match upstream.